analyzers/src/SonarAnalyzer.VisualBasic/Rules/UseReturnStatement.cs

```diff
@@ -54,10 +54,10 @@ public override void VisitIdentifierName(IdentifierNameSyntax node)
             {
                 if (IsImplicitReturnValue(node))
                 {
-                    context.ReportIssue(Diagnostic.Create(Rule, node.GetLocation(),
+                    context.ReportIssue(Rule, node,
```

Comment:
Please fix the formatting on this tone. Single line should do the trick . Otherwise, `Rule` and `node` would need separate lines, while the ternary should be on a single line